giskard/llm/generators/base.py

```diff
@@ -31,11 +33,16 @@ def __init__(
         llm_temperature: Optional[float] = None,
         llm_client: LLMClient = None,
         prompt: Optional[str] = None,
+        languages: list[str] = None,
     ):
         self.llm_temperature = llm_temperature if llm_temperature is not None else self._default_temperature
         self.llm_client = llm_client or get_default_client()
+        self.languages = languages if languages is not None else []
         self.prompt = prompt if prompt is not None else self._default_prompt
 
+        if isinstance(self.languages, list) and self.languages != []:
```

Comment:
I am not sure about this, we need to check that self.languages is actually a list otherwise it won't generate a correct prompt. I would keep it like this: 
```
if self.languages and isinstance(self.languages, list):
```